src/client/src/api/fetchApiV2.js

```diff
@@ -34,18 +34,28 @@ export async function fetchApiV2(url, method, payload = null) {
   }
 }
 
+// layers
 export const fetchLayerById = async id =>
   await fetchApiV2(`layer/${id}`, "GET");
 
 export const fetchLayersByProfileId = async profileId =>
   await fetchApiV2(`layer?profileId=${profileId}`, "GET");
 
+export const updateLayer = async layer => {
+  // remove derived objects
+  delete layer.createdBy;
+  delete layer.updatedBy;
+  return await fetchApiV2(`layer`, "PUT", layer);
```

Comment:
PP: Überall die gleichen Anführungs- und Schlusszeichen verwenden
```suggestion
  return await fetchApiV2("layer", "PUT", layer);
```